.circleci/config.yml

```diff
@@ -39,3 +39,6 @@ workflows:
       - jest-ui-components:
           requires:
             - setup
+      - jest-ui-components-a11y:
```

Comment:
You may want to remove this in lieu of the other PR since it fails w/out the updates